shard-operator-parent/shard-operator-v2/src/test/java/com/redhat/service/smartevents/shard/operator/v2/ManagerClientTest.java

```diff
@@ -55,4 +66,21 @@ public void TestFetchProcessorsToDeployOrDelete() throws JsonProcessingException
 
         assertThat(managerClient.fetchProcessorsToDeployOrDelete().await().atMost(Duration.ofSeconds(10)).size()).isEqualTo(1);
     }
+
+    @Test
+    public void TestNotifyBridgeStatus() {
```

Comment:
@vaibhavjainwiz nitpick, but could you please using camel case for method names. Cheers!

Reply:
done